lessons/Early-Math/associative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,123 @@
+* Students may need support in parsing expressions that include both an operation that _does_ support Associativity along side an operation where Associativity does _not_ apply.
+|===
+
+== The Associative Property @duration{30 minutes}
+
+=== Overview
+
+Students explore Circles of Evaluation to discover that no matter how we group numbers being either added or multiplied, the end result will always be the same.
+
+=== Launch
+
+When asked to multiply @math{2 \times 17 \times 5}, the student who notices that @math{2 \times 5 = 10} will arrive at a result much more efficiently than a student who does not. This is the power of the @vocab{Associative Property}!
+
+In the short term, familiarity with the Associative Property enables students to make mental computations more efficiently. In the long term, comfort with this law of arithmetic (and others!) will provide a strong foundation for more complex algebraic reasoning.
+
+A note: We view the Associative Property as the @vocab{Commutative Property} *2.0*! With the Associative Property, students can not only change the order of the values in a multiplication or addition expression... they can change the _groupings_. _Finding the best groupings, students will discover, sometimes requires changing the order as well._
+
+[.lesson-instruction]
+- Turn to @printable-exercise{assoc-prop-launch.adoc}.
+- Look at the expression at the top of the page: @math{88 + 71 + 637 + 129 + 12}
+- How many different ways can you re-group and re-order the expression? Draw as many equivalent circles of evaluation as you can in the boxes below, then answer the questions at the bottom of the page.
+- As you complete the Circles of Evaluation, strive for groupings that will make the computation simpler.
+
+Allow students to work, encouraging them to try not only different groupings but different _structures_ as well - all of which will produce the same outcome! If possible, invite students to draw their Circles of Evaluation on the board, so that the class can see all possible Circles. After students have drawn their Circles of Evaluation and had some time to consider the reflection questions, follow-up with a class discussion.
+
+[.lesson-instruction]
+- Which Circle of Evaluation seemed like it would be the most difficult to solve in your head? Why?
+- Which Circle of Evaluation seemed like it would be the easiest to solve in your head? Why?
+- Did you think that any Circles of Evaluation were _equally_ easy or difficult to evaluate? What was going on in those Circles of Evaluation?
+** _Students who applied the Commutative Property - merely changing the order of two numbers in the same group - would encounter Circles of Evaluation that were equally easy / challenging to evaluate._
+
+
+=== Investigate
+
+Explain to students that they have just made use of the @vocab{Associative Property} of Addition!
```

Comment:
I don't think that we can have students do a workbook page on regrouping without having already explicitly acknowledged that the Associative property exists. This gets to your question on how to start the lesson. If you want so many groupings to be the launch you have to declare the property (which feels ok to me, particularly if couched as an extension of the commutative property) before you have them work on that page.